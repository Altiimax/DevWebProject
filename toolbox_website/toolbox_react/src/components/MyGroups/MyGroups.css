
#myGroupProfile {
  display: flex;
  flex-direction: column;
}

#newGroupBtn {
  width: 10em;
  margin: 1em 0;
  border-radius: 5px;
  border: solid 2px rgb(155, 152, 152);
  background-color: rgb(18, 204, 18);
  outline: none;
}

#newGroupBtn:hover {
  background-color: rgb(58, 245, 58);;
}

.tableHead {
  display: inline-block;
  position: top;
}

#tbHGroupAdmin{
  width: 3em;
}

#tbHGroupName, #tbHGroupLocation {
  width: 6em;
}

#tbHGroupType{
  width: 3em;
}



/*
.sortButtons {
  display: inline-block;
  margin-left: 5px;
}
.sortButtons span {
  display: block;
  color: white;
}
*/