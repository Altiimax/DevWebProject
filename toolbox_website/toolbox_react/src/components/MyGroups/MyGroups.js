import React, { Component } from "react";
import { Table } from "react-bootstrap";

export default class MyGroups extends Component {
  render() {
    return (
      <div>
        <Table striped bordered hover variant="dark">
          <thead>
            <tr>
              <th>Name of the group</th>
              <th>Location</th>
              <th>Number of member</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>Test</td>
              <td>Test</td>
              <td>Test</td>
            </tr>
            <tr>
              <td>Test</td>
              <td>Test</td>
              <td>Test</td>
            </tr>
          </tbody>
        </Table>
      </div>
    );
  }
}
