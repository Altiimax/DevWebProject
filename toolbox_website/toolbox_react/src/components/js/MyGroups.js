import React, { Component } from "react";

export default class MyGroups extends Component {
  render() {
    return <div>Here will be displayed my groups!</div>;
  }
}
