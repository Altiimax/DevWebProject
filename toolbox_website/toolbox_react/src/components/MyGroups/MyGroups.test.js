import React from "react";
import ReactDOM from "react-dom";
import MyGroups from "./MyGroups.js";
import TestRenderer from "react-test-renderer";
it("render without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<MyGroups />, div);
});

/*
Test de la snapshot (a faire que si le component ne bouge pas trop dans le temps)
Ce test ajoute un dossier _snaphsot_dans lequel est contenu toutes les snapshots
*/
it("renders a table", () => {
  const testRenderer = TestRenderer.create(<MyGroups />);
  const testInstance = testRenderer.toJSON();
  expect(testInstance).toMatchSnapshot();
});
