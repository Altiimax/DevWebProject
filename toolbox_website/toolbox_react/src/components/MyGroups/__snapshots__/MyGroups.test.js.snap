// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a table 1`] = `
<div>
  <table
    className="table table-dark table-striped table-bordered table-hover"
  >
    <thead>
      <tr>
        <th>
          Name of the group
        </th>
        <th>
          Location
        </th>
        <th>
          Number of member
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          Test
        </td>
        <td>
          Test
        </td>
        <td>
          Test
        </td>
      </tr>
      <tr>
        <td>
          Test
        </td>
        <td>
          Test
        </td>
        <td>
          Test
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
